website/content/docs/secrets/ldap.mdx

```diff
@@ -199,6 +199,10 @@ Some important things to remember when crafting your LDIF entries:
 
 ### Active directory (AD)
 
+-> Note: Active Directory might allow old passwords to be used for authentication during a certain amount of time.
+This is refered to as the `lifetime period of an old password`, this setting is configurable on the Windows Servers hosting Active Directory.
+For more information refer to the following Microsoft [guide](https://learn.microsoft.com/en-us/troubleshoot/windows-server/windows-security/new-setting-modifies-ntlm-network-authentication)
```

Comment:
```suggestion
<Note> 

  Windows Servers hosting Active Directory include a 
  `lifetime period of an old password` configuration setting that lets clients
  authenticate with old passwords for a specified amount of time.

  For more information, refer to the
 [NTLM network authentication behavior](https://learn.microsoft.com/en-us/troubleshoot/windows-server/windows-security new-setting-modifies-ntlm-network-authentication)
  guide by Microsoft.

</Note>
```

Style correction: write in active voice, avoid "this" as a pronoun (copied suggestion from PR #28429)